utils/api/benefitHandler.ts

```diff
@@ -158,7 +161,7 @@ export class BenefitHandler {
       false,
       this.future,
       false,
-      this.input.client.age
```

Comment:
in cases when age from the form is different from age we want to calculate the benefits for (ex. form input 61, we run the benefit handler for both 61, then future ages, for example 65, 72 etc.) we would need to use right age. With the change you added here, it would be 61 for every case.

Reply:
fixed